### ISSUE TITLE
Translate into Persian

### ISSUE THREAD

**Issue by Mehraddev** [expressjs/expressjs.com#1580]
Hello Express.js team,

I'm interested in contributing to the project by translating the documentation into Persian. I believe this could help expand the framework's accessibility to Persian-speaking developers.

Before I begin i wanted to ask if such a contribution would be helpful to the project and if there are any specific guidelines I should follow. If there's already an ongoing effort for this, I'd be happy to collaborate.

Looking forward to your feedback!

**Comment by bjohansebas** [comment]
Hi @Mehraddev here are some guides on how to start - https://github.com/expressjs/expressjs.com/blob/gh-pages/CONTRIBUTING.md#contributing-translations

If you have any questions, feel free to ask. Someone from the team will help you (as soon as we can).

**Comment by Mehraddev** [comment]
is there any way to preview my work? i don't seem to find .html files to view the changes.

**Comment by IamLizu** [comment]
Hi @Mehraddev 👋 

This website is built with Jekyll, which is built with Ruby. A little Google search on "how to install Jekyll" will help you. 

Please read the README.md available at the root of this project, it includes instructions to run the site locally.

**Comment by crandmck** [comment]
@Mehraddev Also, we recently implemented Netlify deploy previews.  So when you open a PR, a preview site will be deployed that implements the change, e.g. https://deploy-preview-1568--expressjscom-preview.netlify.app/ for PR #1568.  You may want to also run/test changes locally, but if not you can at least see them with the deploy preview.

**Comment by IamLizu** [comment]
Hey @Mehraddev I am assigning this to you 🎉 

If you need any _real-time_ help, we are available in the [OpenJS Slack](https://openjsf.org/collaboration).

**Comment by chrisdel101** [comment]
TL;DR  - please use a draft PR for development.

Just a note: If you do end up _not_ running it locally, and rather using the preview feature, the best way for us is if you can please make it a draft pull request during development. Then when you switch it to a full non-draft, we know it's ready for review.

**Comment by bjohansebas** [comment]
We have paused the translations, so I am going to close this issue.
